Splitio.Redis/Services/Client/Classes/RedisClient.cs

```diff
@@ -84,11 +84,17 @@ private void ReadConfig(ConfigurationOptions config)
             RedisConnectRetry = config.CacheAdapterConfig.ConnectRetry ?? 0;
             RedisUserPrefix = config.CacheAdapterConfig.UserPrefix;
             LabelsEnabled = config.LabelsEnabled ?? true;
+            BlockMilisecondsUntilReady = config.Ready ?? 0;
         }
 
         private void BuildRedisCache()
         {
             redisAdapter = new RedisAdapter(RedisHost, RedisPort, RedisPassword, RedisDatabase, RedisConnectTimeout, RedisConnectRetry, RedisSyncTimeout);
+            if (BlockMilisecondsUntilReady > 0 && !redisAdapter.IsConnected())
+            {
+                throw new TimeoutException($"SDK was not ready in {BlockMilisecondsUntilReady} miliseconds");
```

Comment:
SDK was not ready in {BlockMilisecondsUntilReady} miliseconds. Could not connect to Redis"